datetimeparser/evaluatormethods.py

```diff
@@ -124,18 +126,36 @@ def remove_milli_seconds(dt: datetime) -> datetime:
 
     @staticmethod
     def get_offset(con: Constant, offset) -> RelativeDateTime:
+        """
+        Calculates the UTC-offset from a Constant-object
+
+        :param con: the Constant
+        :param offset: the UTC-offset from the timezone
+        :return: RelativeDateTime
+        """
+
         off: int = 0
         if con.offset:
             if con.offset < 0:
                 off += abs(con.offset)
             else:
                 off += con.offset
+
             return RelativeDateTime(hours=off + offset.seconds / 3600 + offset.days * 24)
 
 
 class EvaluatorMethods(EvaluatorUtils):
+    """
+    Evaluates a datetime-object from a given list returned from the parser
+    """
 
     def __init__(self, parsed, current_time: datetime, offset: timedelta = None):
```

Comment:
Add at least some kind of typing to `parsed` please. The returntype will always be a tuple containing the method and the data e.g. `Tuple[Method, Any]`